orix/crystal_map/crystal_map.py

```diff
@@ -331,7 +331,9 @@ def orientations(self):
                 rotations = self.rotations[:, 0]
             else:
                 rotations = self.rotations
-            return Orientation(rotations).set_symmetry(phases[:].point_group)
+            o = Orientation(rotations)
+            o.symmetry = phases[:].point_group
+            return o.compute_symmetry_reduced_orientations()
```

Comment:
I suggest changing the behaviour of `CrystalMap.orientations` and not returning the orientations with the smallest rotation angle, and leaving that up to the user to do. This change should be added to the changelog.

```suggestion
            orientations = Orientation(rotations)
            orientations.symmetry = phases[:].point_group
            return orientations
```

Reply:
Makes sense to me too, I will update the tests to reflect this.